ha/core/system_health/bootstrap/health_event_generator.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import uuid
+from cortx.utils.conf_store import Conf
+from cortx.utils.log import Log
+from ha.execute import SimpleCommand
+
+from ha.core.system_health.const import NODE_MAP_ATTRIBUTES
+from ha.core.system_health.const import RESOURCE_TYPE_MAPPING
+from ha.core.system_health.const import CONFSTORE_KEY_ATTRIBUTES
+from ha.const import _DELIM
+from ha.const import EVENT_ATTRIBUTES
+from ha.core.system_health.const import RESOURCE_TO_HEALTH_STATUS_MAPPING
+from ha.core.system_health.const import HEALTH_STATUS_TO_EVENT_SEVERITY_MAPPING
+from ha.core.system_health.model.health_event import HealthEvent
+from ha.core.system_health.system_health import SystemHealth
+from ha.core.error import HaConfigException
+
+class HealthEventGenerator:
+
+    def __init__(self):
+        """
+        Init method.
+        """
+        self._execute = SimpleCommand()
+        self._machine_id = self._get_machine_id()
+
+    def _get_machine_id(self):
+        command = "cat /etc/machine-id"
```

Comment:
Lets move this command string to const.py file in system health.

Reply:
Done